libs/explorers/services/src/lib/synapse-api.service.ts

```diff
@@ -23,26 +19,13 @@ import { SynapseWiki, TermsOfUseInfo } from '@sagebionetworks/explorers/models';
   providedIn: 'root',
 })
 export class SynapseApiService {
+  http = inject(HttpClient);
+
   private cache: { [key: string]: any } = {
     wikis: {},
     terms: null,
   };
 
-  constructor(
-    private http: HttpClient,
-    private wikiPageService: WikiPageServicesService,
-  ) {}
-
-  // I created this method or to facilitate the comparison with `getWiki()`. In practice, there is
-  // no need for a wrapper method like done here: the Explorer components and services should directly
-  // make call to the Synapse API using the Angular client.
-  getWikiAlternative(
-    ownerId: string,
-    wikiId: string,
-  ): Observable<OrgSagebionetworksRepoModelWikiWikiPage> {
-    return this.wikiPageService.getRepoV1EntityOwnerIdWikiWikiId(ownerId, wikiId);
-  }
-
```

Comment:
Removing getWikiAlternative as it is unused.  This can be added back if using an OpenAPI version is needed/desired in the future.  However, this is the culprit behind high memory utilization and performance issues in Jest tests involving this implementation so this will involve some investigation.  BABEL complains about hitting thresholds above 500kb of styling in .ts files when running unit tests with this implementation.